slacklib_test.py

```diff
@@ -1,70 +1,76 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
 
 import datetime
 import textwrap
 import unittest
+from zoneinfo import ZoneInfo
 
-# Update sys.path so it can find these.  We just need to add
-# 'google_appengine', but we add all of $PATH to be easy.  This
-# assumes the google_appengine directory is on the path.
-import os
-import sys
-sys.path.extend(os.environ['PATH'].split(':'))
-import dev_appserver
-dev_appserver.fix_sys_path()
-from google.appengine.ext import db
 from google.appengine.ext import testbed
+import time_machine
 
 import models
 import slacklib
 
-
+# The fictional day for these tests Wednesday, July 29, 2015
+@time_machine.travel(datetime.datetime(2015, 7, 29, tzinfo=ZoneInfo("UTC")))
 class SlashCommandTest(unittest.TestCase):
```

Comment:
I'm confused about this because you're using pytest now, right?  Does pytest support unittest.TestCase classes?  Did you have a desire to move this to a more idiomatic pytest format?  (Not in this diff, of course, but you could add a TODO if you wanted to.)

Reply:
pytest does support unittest.TestCase classes!  My general plan was to gradually transition the tests to a more idiomatic pytest style, but not all in this diff.

Reply:
That sounds great.  I'd just put in TODOs in the places you want to transition.  But it's up to you.